src/jsi/decorator.h

```diff
@@ -126,6 +126,9 @@ class RuntimeDecorator : public Base, private jsi::Instrumentation {
       const std::shared_ptr<const PreparedJavaScript>& js) override {
     return plain().evaluatePreparedJavaScript(js);
   }
+  bool drainMicrotasks(int /*maxMicrotasksHint*/) {
+    return true;
```

Comment:
If I recall the stub returns false on other platforms. Any reason why this returns true?

Reply:
Actually this I think is normally just part of the JSI package and forwards bits along. See https://github.com/facebook/react-native/blob/1465c8f3874cdee8c325ab4a4916fda0b3e43bdb/ReactCommon/jsi/jsi/decorator.h

Reply:
My memory for the JSC one was wrong. That does return true as well 👍.